e2e/pages/adf/paginationPage.js

```diff
@@ -130,6 +131,16 @@ var PaginationPage = function () {
         Util.waitUntilElementIsNotOnPage(previousButtonDisabled);
     };
 
+    this.getTotalNumberOfFiles = function () {
+        Util.waitUntilElementIsVisible(totalFiles);
+        var numberOfFiles = totalFiles.getText().then(function (totalNumber) {
+          var totalNumberOfFiles = totalNumber.substring(16, totalNumber.length);
```

Comment:
Can you use, instead of '16', the character that is at that position?

Reply:
This method takes the total number of the text "Showing 1-20 of 26415" and in this case the starting point will always remain 16. I can assign the number to a variable, but it still will be 16.

Reply:
If you will be on the second page when you will want to use this method, i think 16 will not work anymore, cause the text will be "Showing 21-40 of 26415". I think it's better to use a character/string instead of the position. In this case, you could split by 'of '.